doc/tvpl/c2-setup.md

```diff
@@ -76,16 +76,16 @@ Now that the dependencies have been installed, let's set up a place to build *Vo
 
 Next, let's get the code for *Volta* (the compiler), and *Watt* (the standard library).
 
-	git clone git@github.com:VoltLang/Volta.git
-	git clone git@github.com:VoltLang/Watt.git
+	git clone https://github.com/VoltLang/Volta.git
+	git clone https://github.com/VoltLang/Watt.git
```

Comment:
Permission denied occurs at clone for users who do not register keys in github, so https may be better ;)

```
$ git clone git@github.com:VoltLang/Volta.git
Cloning into 'Volta'...
Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```